propelauth_py/api/org.py

```diff
@@ -65,11 +65,22 @@ def _fetch_org_by_query(
 ####################
 #       POST       #
 ####################
-def _create_org(auth_url, integration_api_key, name, max_users=None):
-    url = auth_url + f"{ENDPOINT_PATH}/"
-    json = {"name": name}
-    if max_users is not None:
-        json["max_users"] = max_users
```

Comment:
Note: `max_users` did not seem to exist anymore in the request in the backend, hence it was removed.

Reply:
We should actually just address this by adding max_users to the backend call. I pushed up a PR for it, and we can hold this one back until after we deploy.

Reply:
No problem, I've added it back.